content/en/Platform Deep Dive/Scans/extra_hosts.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Extra Hosts"
+linkTitle: "Extra Hosts"
+weight: 40
+description: >
+  Add extra hosts to include further domains in Web Application scans.
+---
+
+## Description
+
+{{% image src="/deepdive/scans/extra-hosts/ExtraHosts_List.png" alt="Extra Hosts page" %}}
+
+An extra host serves the purpose of including non-origin domains in the scope of a Web target's scan. 
+Oftentimes, a web application will send requests to a domain that's different from where it was loaded from. By default, those interactions won't be taken into account when scanning for vulnerabilities. However, if these domains are specified as extra hosts, they will be included by the scanner.
```

Comment:
I think an example could help clarifying what is a "host" (specially in regards of subdomains) and in which cases is useful. In Probely they mention SPAs and API's for instance https://help.probely.com/en/articles/1975501-what-are-the-extra-hosts-in-scope-for

Reply:
Good one, will add the example mentioned in that link.